core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -84,12 +83,12 @@ abstract protected[controller] class RestAPIVersion(
 protected[controller] object RestAPIVersion_v1 {
     def requiredProperties =
         WhiskConfig.whiskVersion ++
-            WhiskServices.requiredProperties ++
+            WhiskAuthStore.requiredProperties ++
```

Comment:
I also thought it makes sense to have the properties required local to where the corresponding objects are created, which is in RestAPIs